.circleci/config.yml

```diff
@@ -23,8 +23,8 @@ jobs:
           env: RUST_BACKTRACE=1
           command: cargo test --locked
       - run:
-          name: Ensure schema generation works
-          command: cargo schema
+          name: Build and run schema generator
+          command: cargo schema --locked
```

Comment:
This is good. I would be interested in adding a command to ensure this doesn't changes anything either (to be tested on escrow/erc20 first). Something like following this with `git diff schema` (but that only outputs the diff, if any, doens't have a useful return code)

Reply:
okay, will do that. I have a command somewhere ready for copy/paste.

Reply:
Done in https://github.com/confio/cosmwasm-examples/pull/13 and https://github.com/confio/cosmwasm-template/pull/8/commits/26102761138ed83e23e9bb7a669a0a805a426171